fastparse/shared/src/test/scala/fastparse/ByteTests.scala

```diff
@@ -266,37 +267,579 @@ object ByteTests extends TestSuite {
       }
     }
     'example{
+      'bs{
+        import fastparse.byte._
+
+        assert(BS(0x01, 0xff).toSeq == Array[Byte](0x01.toByte, 0xff.toByte).toSeq)
+        assert(BS("hello":_*).toSeq == Array[Byte](104, 101, 108, 108, 111).toSeq)
```

Comment:
Hmm, I missed this conversion from char* to bytes*. Can you explain it more detailed?